.github/workflows/ansible-managed-var-comment.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: Check for ansible_managed variable use in comments
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+  push:
+    branches:
+      - main
+      - master
+  workflow_dispatch:
+jobs:
+  ansible_managed_var_comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+      - name: Install tox, tox-lsr
+        run: |
+          set -euxo pipefail
+          python -m pip install --upgrade pip
```

Comment:
maybe let's explicitly use `python3` and `pip3`, so we won't get problems when new versions of ubuntu won't ship those symlinks by default anymore

(this applies also to all the other actions added in this PR)